### DISCUSSION TITLE
DynamicDialog component events aren't caught

### DISCUSSION THREAD

**Discussion by andreybavt** [primefaces/community#1228] [PrimeVue]
the snippets shown here:
https://primevue.org/dynamicdialog/#events

do not work, an event isn't intercepted by the dialog

Tested on PrimeVue 3.48.1

**Comment by alkgeopa** [comment]
Can confirm, I'm on PrimeVue 3.50 and it doesn't seem to work either.

**Comment by iprinter4** [comment]
Can confirm too, also on 3.50. Component is emitting event , yet, parent doesn't receive  it. Currently show stopper for me :-(
